ci-automation/vendor-testing/aws.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Copyright (c) 2022 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -euo pipefail
+
+# Test execution script for the AWS vendor image.
+# This script is supposed to run in the mantle container.
+
+source ci-automation/vendor_test.sh
+
+board="${CIA_ARCH}-usr"
+image_name="ci-${CIA_VERNUM//+/-}"
+aws_instance_type_var="AWS_${CIA_ARCH}_INSTANCE_TYPE"
+aws_instance_type="${!aws_instance_type_var}"
+more_aws_instance_types_var="AWS_${CIA_ARCH}_MORE_INSTANCE_TYPES"
+set -o noglob # there shouldn't be any instance types with asterisks
+              # in it, but…
+more_aws_instance_types=( ${!more_aws_instance_types_var} )
+set +o noglob
+
+vmdk='flatcar_production_ami_vmdk_image.vmdk'
+
+if [[ "${AWS_AMI_ID}" == "" ]]; then
+    if [[ -f "${vmdk}" ]]; then
+        echo "++++ ${CIA_TESTSCRIPT}: using existing ${vmdk} for ${CIA_VERNUM} (${CIA_ARCH}) ++++"
+    else
+        echo "++++ ${CIA_TESTSCRIPT}: downloading ${vmdk}.bz2 for ${CIA_VERNUM} (${CIA_ARCH}) ++++"
+        copy_from_buildcache "images/${CIA_ARCH}/${CIA_VERNUM}/${vmdk}.bz2" .
+        lbunzip2 "${vmdk}.bz2"
```

Comment:
Since we have the problem with the disk space, how about making this file sparse by trying something like this:
```
cp --sparse=auto <(lbzcat "${vmdk}.bz2") "${vmdk}"
rm "${vmdk}.bz2"
```

Reply:
(I meant proposing this for Azure and this is the AWS file, anyway, Sayan picked it up: https://github.com/flatcar-linux/scripts/pull/274/commits/b9bbda63ee1df1f57a39d523897686a1fe5dd677)

Reply:
And funny observation: with btrfs and FS compression this wouldn't have bubbled up even though it is not as efficient as sparse files but zeros compress quite ok.